src/patterns/page-not-found-pages/index.md.njk

```diff
@@ -55,22 +55,19 @@ Do not use:
 - informal or humourous words like oops
 - red text to warn people
 
-### Service mistake
+### If you do not know what caused the page not found
 
 {{ example({group: "patterns", item: "page-not-found-pages", example: "default", html: true, size: "xl"}) }}
 
-### User mistake
+### If you know the page not found is because of a broken link or button inside the service
```

Comment:
I suggest removing this example because I'm not confident that services will be able to reliably know whether the error is the result of a broken link or button inside the service. Also, in that instance, it would be better for the service to prioritise fixing the broken link or button than showing this page.

Reply:
I have spoken to 5 developers and they said it may be possible but you would be guessing. Most services will track 404s and have acceptance tests to make sure they do not happen from within the service so that is what the documentation emphasises.